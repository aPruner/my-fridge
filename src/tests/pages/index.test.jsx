import React from 'react';
import { shallow } from 'enzyme';
import IndexPage from '../../pages/index';

test('should render index page correctly', () => {
  const wrapper = shallow(<IndexPage />);
  expect(wrapper).toMatchSnapshot();
});
