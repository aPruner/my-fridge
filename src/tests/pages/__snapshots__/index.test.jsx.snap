// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render index page correctly 1`] = `
<Layout>
  <SEO
    title="Home"
  />
  <h1>
    Hi people
  </h1>
  <p>
    Welcome to your new Gatsby site.
  </p>
  <p>
    Now go build something great.
  </p>
  <div
    style={
      Object {
        "marginBottom": "1.45rem",
        "maxWidth": "300px",
      }
    }
  >
    <Image />
  </div>
</Layout>
`;
